Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang:1.15
+
+WORKDIR /go/src/app
+COPY . .
+
+RUN go get -d -v ./...
+RUN go install -v ./...
+
+CMD ["tsm"]
```

Comment:
I think this should be done as multistage build. There is no point for golang to be available on container with our app. golang is only needed to build the app.

Golang applications can even be created as simple (and small) as:

```Dockerfile
FROM scratch
COPY
CMD/ENTRYPOINT
```